modules/dnn/test/test_tokenizer.cpp

```diff
@@ -0,0 +1,105 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace  {
+
+template<typename tstring>
+static std::string _tf_gpt2(tstring filename) {
+    std::string basetestdir = getOpenCVExtraDir();
+    size_t len = basetestdir.size();
+    if (len > 0 && basetestdir[len-1] != '/' && basetestdir[len-1] != '\\')
+        return (basetestdir + "/testdata/dnn/llm/gpt2") + filename;
+    return (basetestdir + "testdata/dnn/llm/gpt2/") + filename;
```

Comment:
The `OPENCV_TEST_DATA_PATH` is set to `/path/to/opencv_extra/testdata` in the test system. Please, drop `/testdata` from the path.